README.md

```diff
@@ -580,6 +576,106 @@ This document is written to capture the conventions of writing styles for our pr
 ### Disclaimer
 It's important to note that client and project requirements always undo choices made in these guidelines
 
+### Formatting and Naming
+
+Names should be meaningful and not presentational:
+```css
+/* Bad examples */
+
+/* presentational with color and position reference */
+.button-orange {}
+.left-block {}
+
+/* meaningless */
+.yee-u88 {}
+```
+Some good examples would be:
+```css
+.video {}
+
+.gallery {}
+
+```
+
+Selectors are placed on separate lines:
+```css
+/* Recommended */
+h1,
+h2,
+h3 {
+  text-transform: uppercase;
+}
+
+/* Not recommended */
+h1, h2, h3 {
+  text-transform: uppercase;
+}
+```
+
+Don't use ids as a selector (unless there is no other option)
+
+Don't combine element types with classes and ids. Preferably don't style on elements at all. Reasonable exceptions are in a base reset or when styling elements that can not be targeted in another way (e.g. elements that are generated from a wysiwyg editor or coming from external sources).
+```scss
+/* Recommended */
+.overview {}
+
+/* Encouraged exceptions */
+.wysiwyg {
+  h1,
+  h2,
+  h3 {
+    text-transform: uppercase;
+  }
+}
+
+/* Not recommended */
+ul.overview {}
+
+.overview li {}
+```
+
+Selectors are separated by empty lines
+```css
+/* Recommended */
+.foo {
+  color: #f00;
+}
+
+.bar {
+  color: #0f0;
+}
+
+.baz {
+  color: #00F;
+}
+
+/* Not recommended */
+.foo {
+  color: #f00;
+}
+.bar {
+  color: #0f0;
+}
+.baz {
+  color: #00F;
+}
+```
+
+Properties have their own unique line
+
+```css
+/* Recommended */
+.gallery {
+  background: #000;
+  color: #fff;
+}
+
+/* Not recommended */
```

Comment:
I just want to make sure that the intent is clear here: by writing _Not recommended_, we are saying that we don't recommend this, but we still allow it. Are we okay, with allowing formats that we do not recommended?

Reply:
I made it more clear that certain format styles are bad instead of not recommended